AK/Bitmap.h

```diff
@@ -50,6 +50,36 @@ class Bitmap {
         memset(m_data, value ? 0xff : 0x00, size_in_bytes());
     }
 
+    int find_last_unset()
+    {
+        int i = 0;
+        int j = 0;
+
+        while (m_data[j] == 0x00)
+            j++;
+
+        for (i = j * 8; i < m_size; i++)
+            if (get(i))
+                return i;
+
+        return 0;
```

Comment:
What if the very first bit in the set is 1? Then we'll also return 0. -1 might make more sense :)